hail_scripts/reference_data/config.py

```diff
@@ -272,8 +273,20 @@ def custom_gnomad_select_v3(ht):
     },
     'screen': {
         '38': {
-            'path': 'gs://seqr-reference-data/GRCh38/ccREs/GRCh38-ccREs.ht',
+            'path' : 'gs://seqr-reference-data/GRCh38/ccREs/GRCh38-ccREs.ht',
             'select': {'region_type': 'target'},
+            'enum_select': {
```

Comment:
To get camelCase-ing here, we'd need to modify this to be:

```
'select': {'regionType': 'target'},
'enum_select': {'regionType: ['CTCF-bound',...]},
```

which is straightforward... but leaving as is would allow us backwards compatibility w/ v02.

Reply:
Yeah I think keeping as is makes sense and then we maybe do a single pass of the repo and move to camelCase, similar to what we should do with black/adding docstrings to all funtions? I don't know what the best approach is for supporting v02 vs slightly cleaner v03 repo?